tests.py

```
@@ -0,0 +1,35 @@
+import unittest
+from fibo import fibonacci, geometric_series_aprox
+
+class TestFibonacci(unittest.TestCase):
+    def test_fibonacci_sequence(self):
```

Ruff

Missing return type annotation for public function `test_fibonacci_sequence`

Add return type annotation: `None`

(ANN201)